libs/application/templates/institution-collaboration/README.md

```diff
@@ -0,0 +1,46 @@
+# Institution Collaboration
+
+This application helps institutions to request Stafrænt Ísland to collaborate on a new application.
+
+## Application flow:
+
+- Application has 2 states, DRAFT & APPROVED
+- When applicant submits his application the state automacally changes to APPROVED.
+  - Stafrænt Ísland's contact personnel recevies the application in email, and manually reviews it.
+  - The applicant receives a copy of the application in email but is also able to revisit the application and view the summary page at any time.
+
+## Running unit tests
+
+Run `nx test application-templates-institution-collaboration` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Setup
+
+To start the application system, follow the instructions in the handbook [here](https://docs.devland.is/handbook/apps/application-system).
+
+- Local application URL: [http://localhost:4242/umsoknir/samstarf](http://localhost:4242/umsoknir/samstarf)
+
+#### Configure AWS
```

Comment:
You can remove this part, there is a [PR](https://github.com/island-is/island.is/pull/3295) coming for a more detailed guide